src/jit-pintool/clrjit_inscount.cpp

```diff
@@ -0,0 +1,249 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is this the appropriate license to include given that it is based on Intel's source code?

Reply:
I'm not gonna deny that I have used some snippets from their examples (mostly function declarations), but most of this pintool is written from scratch. Also both are MIT licensed.